ultra/docs/getting_started.md

```diff
@@ -0,0 +1,116 @@
+
+## Generating Scenarios for a Task
+
+To begin, start by generating scenarios for a left-turn task. Each task has various left-turn scenarios that aim to teach the agent something. Task 1 trains the agent to generalize its knowledge on the type of intersection. This is done by creating left-turn scenarios in t-intersections for training, and left-turn scenarios in cross-intersections for testing. All supported tasks are listed under `ultra/scenarios/`.
+
+Task 1 Training Scenario|Task 1 Testing Scenario
+:----------------------:|:---------------------:
+<img src="_static/task1_train.png" width="340" height="220"/> | <img src="_static/task1_test.png" width="340" height="220"/>
+
+- If you have not yet generated the maps used by the ULTRA scenarios (available in `ultra/scenarios/pool/`), run the following command:
+  ```sh
+  $ scl scenario build-all ultra/scenarios/pool
+  ```
+  > Maps are descriptions of the roads used in the scenarios and only need to be compiled once. They only need to be compiled again if they have been modified.
+- Now we can generate the scenarios for Task 1. Each task has a `config.yaml` file describing the levels of the task:
+  ```yaml
+  ego_mission:
+  ...  # The start and end points of the agent's route.
+  level:
+    ...
+    <level_name>:
+      train:
+        total:  # The number of training scenarios to generate.
+        intersection_types:
+          <intersection_shape>:
+              percent:  # Proportion of scenarios with this intersection.
+              specs:  # Speed, density, and proportion of the traffic.
+      test:
+        ...
+    ...
+  ```
+  Notice in `ultra/scenarios/task1/config.yaml` the levels of Task 1 include "hijack", "no-traffic", "easy", and "hard". Each level has "train" and "test" scenarios defined by the "total" number of scenarios produced and the intersection types. For example, Task 1's "easy" level produces 10000 training scenarios containing all t-intersections, and 200 testing scenarios containing all c-intersections. Modify these numbers to be 10 and 2 respectively,  and then generate the "easy" level scenarios of Task 1:
+  ```sh
+  $ python ultra/scenarios/interface.py generate --task 1 --level easy
+  ```
+  > There should now be 12 (= 10 + 2) scenario folders under `ultra/scenarios/task1/`.
+- (Optional) Now that we are ready to train and evaluate, we can start Envision to visualize the process. To do this, run the following command:
+  ```sh
+  $ ./ultra/env/envision_base.sh
+  ```
+  > Envision runs as a background process, you can view the visualization on `localhost:8081/`.
+
+## Training a Baseline Agent
+
+Implementations of baseline agents are available in `ultra/baselines/`. Notice, policies such as PPO, SAC, DDPG, and DQN are implemented as baselines. We will run a DQN on Task 1's "easy" level in this example.
+
+- First, in `ultra/train.py`, change the following line:
```

Comment:
This is not relevant to the docs, but maybe this should also be a flag with a default set.

Reply:
Yeah, there is currently, unfortunately, no way to set a flag for the policy you want to train/evaluate. I think Jenish is working on something though. I thought maybe because there is no intuitive way to specify the agent you want to train/evaluate, it might be good to use an example? But should this be made more general?

Reply:
Something like this could work though,

```
    parser.add_argument(
        "--policy_class", help="environment seed", default="ultra.baselines.sac:sac-v0", type=str,
    )
```

Reply:
Oh sure yeah, that sounds good; will implement this.

Reply:
We can maybe implement this after the docs are merged, just so that this PR is focussed purely on documentation.